knora/dsplib/models/group.py

```diff
@@ -240,23 +240,18 @@ def delete(self):
         return Group.fromJsonObj(self._con, result['group'])
 
     @staticmethod
-    def getAllGroups(con: Connection) -> Optional[list[Group]]:
-        try:
-            result = con.get(Group.ROUTE)
-            return [Group.fromJsonObj(con, group_item) for group_item in result["groups"]]
-        except BaseError:
-            # return None if no groups are found or an error happened
-            return None
```

Comment:
I want the error to escalate, so that I can handle it in the higher-level methods